dogpile/cache/backends/redis.py

```diff
@@ -95,13 +97,16 @@ class RedisBackend(BytesBackend):
 
      .. versionadded:: 1.1.6  Added ``connection_kwargs`` parameter.
 
+     .. versionadded:: 1.3,2  Added ``username`` parameter.
```

Comment:
write it like this:

    .. versionadded:: 1.3.1  Added ``username`` parameter.

Reply:
Fixed in the latest version.